src/ol/renderer/canvas/IntermediateCanvas.js

```diff
@@ -93,20 +92,7 @@ class IntermediateCanvasRenderer extends CanvasLayerRenderer {
    * @inheritDoc
    */
   forEachFeatureAtCoordinate(coordinate, frameState, hitTolerance, callback, thisArg) {
-    const layer = this.getLayer();
-    const source = layer.getSource();
-    const resolution = frameState.viewState.resolution;
-    const rotation = frameState.viewState.rotation;
-    const skippedFeatureUids = frameState.skippedFeatureUids;
-    return source.forEachFeatureAtCoordinate(
-      coordinate, resolution, rotation, hitTolerance, skippedFeatureUids,
-      /**
-       * @param {import("../../Feature.js").FeatureLike} feature Feature.
-       * @return {?} Callback result.
-       */
-      function(feature) {
-        return callback.call(thisArg, feature, layer);
-      });
+    return undefined;
```

Comment:
Do `@abstract` methods need to be implemented by every subclass? If not, this function could be removed.

Reply:
You're right, this can be removed. It's functionally no different than the abstract class and is not needed here.